SimpleXML/SimpleXML.php

```diff
@@ -358,7 +358,7 @@ function simplexml_load_file ($filename, $class_name = "SimpleXMLElement", $opti
  * <b>TRUE</b> if <i>ns</i> is a prefix, <b>FALSE</b> if it's a URI;
  * defaults to <b>FALSE</b>.
  * </p>
- * @return SimpleXMLElement an object of class SimpleXMLElement with
+ * @return mixed an object of class SimpleXMLElement with
```

Comment:
`SimpleXMLElement|false`.

Some PhpStorm meta magic should be applied to gather return type from `$className` parameter.